changelog.d/scriv.ini

```diff
@@ -0,0 +1,5 @@
+[scriv]
```

Comment:
Same remark as in ggshield : should we store all these config in `pyproject.toml` ?